org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/JavaProtocolExtensions.java

```diff
@@ -35,4 +35,14 @@
 	 */
 	@JsonNotification
 	void projectConfigurationUpdate(TextDocumentIdentifier documentUri);
+
+	/**
+	 * Request to load/unload a 3rd-party bundles.
+	 *
+	 * @param param
+	 *            the parameters passed to operation.
+	 * @return the result of bundle operation
+	 */
+	@JsonRequest
+	CompletableFuture<Object> bundle(BundleRequestParams param);
```

Comment:
IMO, it would make more sense to add two protocol extensions: "install" and "uninstall" with properly typed parameter classes.  
Also, I would have to look at the implementation to know what values I have to pass into this method and to know possible result values. Systems should be understandable by their interface, not their implementation.